apps/jscad-web/README.md

```diff
@@ -27,3 +27,18 @@ To start the production server run:
 ```
 npm run serve
 ```
+
+# using url to load external script and CORS
+
+If you want to share a script from your website you need to setup CORS, and make sure to use HTTPS!
```

Comment:
nit: this is only true if you're running in dev mode, or statically hosted copy of the app. If it's running in serve mode, it can fallback to the /remote endpoint to bypass CORS.

Reply:
I updated the description and added a disclaimer

if you do not setup CORS [jscad.app](https://jscad.app) can fallback to `/remote` to download the script, but this workaround  may not be available forever (such enpoint could be abused to hide IP for attacks).